### DISCUSSION TITLE
Microphone/Speaker API

### DISCUSSION THREAD

**Discussion by adripc64** [zepp-health/zeppos-forum#204] [Q&A]
Hello, I would like to know if I can use the microphone/speaker capabilities of the device in a mini program. I can't find corresponding API in the docs. If these features are not supported yet, is there any plan to add them in future versions?

**Comment by liqin-ZeppHealth** [comment]
hello @adripc64 , we are going to open APIs for Microphone & Speaker in Zepp OS3.0, you can talk with @MyoungXUE or Juanita on slack to join the 3.0 beta development

**Reply by adripc64** [reply]
Thank you @liqin-ZeppHealth for you answer.

That's nice, I hope we can use these capabilities on 3.0 version. Can I assume that devices like GTR/GTS 4 and the new Amazon Cheetah(PRO) will also get such version?

BTW, I can't join to the slack community as I don't have a @zepp.com email, could I get an invitation?

**Reply by liqin-ZeppHealth** [reply]
3.0 is still a beta version for new products. We also want to upgrade the new version on as many devices as possible. but I'm afraid it needs time and there is no clear plan for it by now.
we will send you slack invitation, welcome to the community!

**Reply by liqin-ZeppHealth** [reply]
can you try to join this: https://t.zepp.com/t/zepp-community-slack

**Reply by adripc64** [reply]
I already tried that, but it seems it requires an email with @zepp.com domain:
![image](https://github.com/zepp-health/zeppos-forum/assets/5053228/525b605c-2cf2-47e9-adae-fd04c62390f0)

If I try to signup with my gmail account:
![image](https://github.com/zepp-health/zeppos-forum/assets/5053228/05c28251-7d25-4431-81fd-30196a787c38)

**Reply by MyoungXUE** [reply]
In theory, clicking Google can be successful, it is recommended to try again.

If that doesn't work, you can log in to Slack at https://slack.com/ and click this link https://t.zepp.com/t/zepp-community-slack again

**Reply by adripc64** [reply]
No way, If I try to login with google the error in the screenshote above is shown. It requires me to have an email with @zepp.com domain.
